package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "main": "./index.js",
   "types": "./index.d.ts",
+  "type": "commonjs",
```

Comment:
Does it have to be commonjs? I think if the file has extension .mjs it will be interpreted accordingly.

Reply:
Oh, I forgot that the whole thing is commonjs. It would have been nice to publish new packages as ESM only…

Reply:
I don’t perceive `const { method } = await import('module')` as complex, do you?

Reply:
We have copied the source code of multiple ESM libraries to the sdk because we couldn't get it to work. How about you fix that and then we talk again.
It starts with every function using these imports becoming async, adds the extra weirdness of imports not being global any more (you need to add that to every function that uses the function), and the cherry on top is that you have to find a way to convince typescript not to turn the `import()` into a `require()`, which for some stupid reason it does in all but very specific configurations.

Reply:
Yeah, I guessed I rushed my conclusions. Sorry, I was too upset, seeing the step back from modernity.

Reply:
We do think about dropping CJS support throughout our libraries though, so this is hopefully temporary. But keep in mind that you'd have to change SKYC's configuration, and we still need to figure out if we can convince jest to work properly with it